pages/reset-password.js

```diff
@@ -119,7 +120,7 @@ class Login extends Component {
                   <FontAwesomeIcon icon="user" fixedWidth />
                 </label>
 
-                <input
+                <ValidatedInput
```

Comment:
Shouldn't this be a `PasswordInput`?
If so make sure to include the props detailed on the registration page comment.